torch/csrc/cuda/nccl.cpp

```diff
@@ -25,6 +25,10 @@
 #define NCCL_HAS_COMM_NONBLOCKING 1
 #endif
 
+#if (NCCL_MAJOR > 2) || (NCCL_MAJOR == 2) && (NCCL_MINOR >= 28)
+#define NCCL_HAS_COMM_NONBLOCKING
```

Comment:
Uh, this is defined on line 25, isn't it?

Reply:
opps rebase error.